tests/aspnetcore/unit-tests.failed

```diff
@@ -1,72 +1,31 @@
-/aspnetcore/artifacts/bin/Microsoft.AspNetCore.Antiforgery.Test/Debug/net6.0/Microsoft.AspNetCore.Antiforgery.Test.dll
-/aspnetcore/artifacts/bin/Microsoft.AspNetCore.ApiAuthorization.IdentityServer.Tests/Debug/net6.0/Microsoft.AspNetCore.ApiAuthorization.IdentityServer.Tests.dll
-/aspnetcore/artifacts/bin/Microsoft.AspNetCore.Authentication.AzureAD.UI.Test/Debug/net6.0/Microsoft.AspNetCore.Authentication.AzureAD.UI.Test.dll
-/aspnetcore/artifacts/bin/Microsoft.AspNetCore.Authentication.AzureADB2C.UI.Test/Debug/net6.0/Microsoft.AspNetCore.Authentication.AzureADB2C.UI.Test.dll
-/aspnetcore/artifacts/bin/Microsoft.AspNetCore.Authentication.Negotiate.Test/Debug/net6.0/Microsoft.AspNetCore.Authentication.Negotiate.Test.dll
-/aspnetcore/artifacts/bin/Microsoft.AspNetCore.Components.Authorization.Tests/Debug/net6.0/Microsoft.AspNetCore.Components.Authorization.Tests.dll
-/aspnetcore/artifacts/bin/Microsoft.AspNetCore.Components.Server.Tests/Debug/net6.0/Microsoft.AspNetCore.Components.Server.Tests.dll
-/aspnetcore/artifacts/bin/Microsoft.AspNetCore.Components.Tests/Debug/net6.0/Microsoft.AspNetCore.Components.Tests.dll
 /aspnetcore/artifacts/bin/Microsoft.AspNetCore.Components.WebAssembly.Authentication.Tests/Debug/net6.0/Microsoft.AspNetCore.Components.WebAssembly.Authentication.Tests.dll
 /aspnetcore/artifacts/bin/Microsoft.AspNetCore.Components.WebAssembly.Server.Tests/Debug/net6.0/Microsoft.AspNetCore.Components.WebAssembly.Server.Tests.dll
 /aspnetcore/artifacts/bin/Microsoft.AspNetCore.Components.WebAssembly.Tests/Debug/net6.0/Microsoft.AspNetCore.Components.WebAssembly.Tests.dll
 /aspnetcore/artifacts/bin/Microsoft.AspNetCore.ConcurrencyLimiter.Tests/Debug/net6.0/Microsoft.AspNetCore.ConcurrencyLimiter.Tests.dll
-/aspnetcore/artifacts/bin/Microsoft.AspNetCore.CookiePolicy.Test/Debug/net6.0/Microsoft.AspNetCore.CookiePolicy.Test.dll
 /aspnetcore/artifacts/bin/Microsoft.AspNetCore.DataProtection.EntityFrameworkCore.Test/Debug/net6.0/Microsoft.AspNetCore.DataProtection.EntityFrameworkCore.Test.dll
-/aspnetcore/artifacts/bin/Microsoft.AspNetCore.DataProtection.Extensions.Tests/Debug/net6.0/Microsoft.AspNetCore.DataProtection.Extensions.Tests.dll
 /aspnetcore/artifacts/bin/Microsoft.AspNetCore.DataProtection.Tests/Debug/net6.0/Microsoft.AspNetCore.DataProtection.Tests.dll
-/aspnetcore/artifacts/bin/Microsoft.AspNetCore.Diagnostics.HealthChecks.Tests/Debug/net6.0/Microsoft.AspNetCore.Diagnostics.HealthChecks.Tests.dll
-/aspnetcore/artifacts/bin/Microsoft.AspNetCore.Diagnostics.Tests/Debug/net6.0/Microsoft.AspNetCore.Diagnostics.Tests.dll
-/aspnetcore/artifacts/bin/Microsoft.AspNetCore.HostFiltering.Tests/Debug/net6.0/Microsoft.AspNetCore.HostFiltering.Tests.dll
-/aspnetcore/artifacts/bin/Microsoft.AspNetCore.Http.Connections.Tests/Debug/net6.0/Microsoft.AspNetCore.Http.Connections.Tests.dll
 /aspnetcore/artifacts/bin/Microsoft.AspNetCore.Http.Extensions.Tests/Debug/net6.0/Microsoft.AspNetCore.Http.Extensions.Tests.dll
```

Comment:
At least this one should pass with removal of `DOTNET_SYSTEM_GLOBALIZATION_INVARIANT`. Please incorporate the change and rerun the tests.